### PR TITLE
[project-base] fixed entity extension test

### PR DESCRIPTION
**Pull request by grossmannmartin**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Entity extension test is now application independent and skipped on project (test is run only in monorepo)
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

----
:globe_with_meridians: Live Preview:
  - https://mg-entity-extension-test.odin.shopsys.cloud
  - https://cz.mg-entity-extension-test.odin.shopsys.cloud

### CODE REVIEW

File: project-base/app/tests/App/Functional/EntityExtension/EntityExtensionTest.php [hunk 1]
```diff
@@ -42,36 +42,33 @@ class EntityExtensionTest extends TransactionFunctionalTestCase
...
+     * @inject
+     */
+    private DatabaseSchemaFacade $databaseSchemaFacade;
+
     protected function setUp(): void
     {
         parent::setUp();
 
-        return;
+        // To ensure the changes in application does not brake this test, it's necessary to start with a clean database
```
Comment (vitek-rostislav) [inline project-base/app/tests/App/Functional/EntityExtension/EntityExtensionTest.php]:
```suggestion
// To ensure the changes in the application do not break this test, it's necessary to start with a clean database
```

File: project-base/app/tests/App/Functional/EntityExtension/EntityExtensionTest.php [hunk 2]
```diff
@@ -42,36 +42,33 @@ class EntityExtensionTest extends TransactionFunctionalTestCase
...
+        // test database is restored thanks to the transactional test case
```
Comment (vitek-rostislav) [inline project-base/app/tests/App/Functional/EntityExtension/EntityExtensionTest.php]:
```suggestion
// test database is restored after the test thanks to the usage of transactional test case
```

File: project-base/app/tests/App/Functional/EntityExtension/Model/Category/Category.php [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+     * @Gedmo\TreeLevel
+     * @ORM\Column(type="integer")
+     */
+    protected int $level;
+
+    /**
+     * @Gedmo\TreeLeft
+     * @ORM\Column(type="integer")
+     */
+    protected int $lft;
```
Comment (vitek-rostislav) [inline project-base/app/tests/App/Functional/EntityExtension/Model/Category/Category.php:71]: I am thinking - do we need so many attributes? In general, in the test entities, IMHO we do not need one type of a column more than once (e.g. in `Category`, I already have an integer column `$level` so `$lft` seems redundant, in `CategoryDomain`, there is `$seoTitle` so `$seoMetaDescription` and other string columns seem unnecessary as well, etc.). 

It seems like there is room for simplifying the test entities

**Review by vitek-rostislav — CHANGES_REQUESTED** [review]

File: project-base/app/tests/App/Functional/EntityExtension/Model/Category/Category.php (hunk 3 above)
Comment (grossmannmartin) [reply project-base/app/tests/App/Functional/EntityExtension/Model/Category/Category.php:71]: good idea, I have removed several columns and kept those I found interesting somehow.

**Review by vitek-rostislav — APPROVED** [review]

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**